rest-helix/src/main/java/com/github/twitch4j/helix/interceptor/TwitchHelixDecoder.java

```diff
@@ -27,11 +28,24 @@ public Object decode(Response response, Type type) throws IOException {
         // track rate limit for token
         String token = singleFirst(response.request().headers().get(AUTH_HEADER));
         if (token != null && token.startsWith(BEARER_PREFIX)) {
-            String remaining = singleFirst(response.headers().get(REMAINING_HEADER));
+            // Parse remaining
+            String remainingStr = singleFirst(response.headers().get(REMAINING_HEADER));
+            Integer remaining;
+            try {
+                remaining = Integer.parseInt(remainingStr);
+            } catch (NumberFormatException ignored) {
+                remaining = null;
+            }
+
+            // Synchronize library buckets with twitch data
             if (remaining != null) {
-                try {
-                    interceptor.updateRemaining(token.substring(BEARER_PREFIX.length()), Integer.parseInt(remaining));
-                } catch (Exception ignored) {
+                String bearer = token.substring(BEARER_PREFIX.length());
+                if (response.request().httpMethod() == Request.HttpMethod.POST && response.request().requestTemplate().path().endsWith("/clips")) {
+                    // Create Clip has a separate rate limit to synchronize
+                    interceptor.updateRemainingCreateClip(bearer, remaining);
```

Comment:
[Create Clip](https://dev.twitch.tv/docs/api/reference#create-clip) is supposed to include `Ratelimit-Helixclipscreation-Limit` & `Ratelimit-Helixclipscreation-Remaining` headers - when testing, these headers weren't present, and twitch simply used the standard rate limit header to express the value (so we simply use `remaining`)

Also worth noting: the moderation APIs with undocumented rate limits do not have any special header behavior, so there is no analogous `updateRemaining` call (as we can't determine endpoint-specific bucket size/remaining)